### DISCUSSION TITLE
Will this be updated for Ranked 2.0?

### DISCUSSION THREAD

**Discussion by Us3r-3C29x** [danielwerg/r6api.js#93] [Q&A]
I just wanted to ask if there are plans to update this api for Ranked 2.0?

^^

**Comment by danielwerg** [comment]
I don't read leaks so I'm not sure what ranked 2.0 will change (please don't post it here :D). I been working on another big re-write as well for few months now but I'm been busy with other live stuff and kinda unmotivated so can't give any ETA on that. But yeah I'm planning to update r6api.js@4 if it's possible without major code changes.
